server/src/utils/metas.ts

```diff
@@ -152,7 +152,9 @@ const encodeWhateverType = (value: NestedMetadata): NestedMetadata => {
 }
 
 const recursiveEncode = (entityMetadata: Metadata): Metadata => {
-  return Object.entries(entityMetadata)
-    .map<[string, NestedMetadata]>(([key, value]) => [key, encodeWhateverType(value)])
-    .reduce((newMetadata, [key, value]) => ({ ...newMetadata, [key]: value }), {})
+  return entityMetadata
+    ? Object.entries(entityMetadata)
+        .map<[string, NestedMetadata]>(([key, value]) => [key, encodeWhateverType(value)])
+        .reduce((newMetadata, [key, value]) => ({ ...newMetadata, [key]: value }), {})
+    : []
```

Comment:
Vu que le type de retour c'est type `Record<string, ...>`
```suggestion
    : {}
```

Reply:
Je ne comprends pas ce que tu veux dire par `Record<string, ...>`, mais si l'array est vide, la map ne joue pas, et Object.entries return un array vide.

```
> Object.entries({})
[]
```

Reply:
On fait un `reduce` sur cet array (ligne 158), du coup ça retourne un object